sunpy/time/time.py

```diff
@@ -13,7 +13,7 @@
 from astropy.time import Time, TimeDelta
 
 # This is not called but imported to register it
-from sunpy.time.utime import TimeUTime  # NOQA
+from sunpy.time.timeformats import TimeUTime  # NOQA
```

Comment:
```suggestion
from sunpy.time.timeformats import *  # NOQA
```
Do we not want to register the new format too here?

Reply:
Yes, but I'm a bit confused as to why this is needed here. I'm already importing it in `time/__init__.py` which should register it already?

Reply:
👍 makes sense

Reply:
Fixed